options/options.css

```diff
@@ -309,4 +309,13 @@ h1,h2,h3,h4,h5,h6 {
 
 svg {
     text-decoration: none; 
+}
+
+.number-container:before {
+    content: attr(label-name);
```

Comment:
Is there a reason this is being set in css?

Reply:
It was copied from https://github.com/ajayyy/SponsorBlock/blob/b5996a9783d2b45993db3235efb6cdbbc864e0c3/options/options.css#L70

Though I do admit I was never 100% sure why it was set in css there either.